docs/References/Runme_Parameters.md

```diff
@@ -0,0 +1,154 @@
+```python
+#in `freemocap/fmc_runme.py`
+def RunMe(sessionID=None,
+        stage=1,
+        useOpenPose=False, 
+        runOpenPose = True, 
+        useMediaPipe=True,
+        runMediaPipe=True,
+        useDLC=False,
+        dlcConfigPath=None,
+        debug=False,
+        setDataPath = False,
+        userDataPath = None,
+        recordVid = True,
+        showAnimation = True,
+        reconstructionConfidenceThreshold = .7,
+        charucoSquareSize = 36, #mm
+        calVideoFrameLength = .5,
+        startFrame = 0,
+        useBlender = False,
+        resetBlenderExe = False,
+      	get_synced_unix_timestamps = True,
+        good_clean_frame_number = 0,
+        bundle_adjust_3d_points=False
+        ):
+```
+
+### [RunMe input parameters](#runme-input-parameters)
+- `sessionID`
+  - Type - (str) 
+  - [Default] - None.
+  - Indentifying string to use for this session. 
+  - If creating a new session, default behavior is to autogerate SessionID is based on date and time that the session was recorded
+  - If re-processing a previously recorded session, this value specifies which session to reprocess (must be the name of a folder within the `FreeMoCap_Data` folder)
+  - 
+- `stage`
+  - [Type] - Int 
+  - [Default] - 1
+  - Which processing stage to start from. Processing stages are deined in more  detail in [#processing-stages](#processing-stages) 
+  
+  ```
+  stage 1 - Record Raw Videos
+  stage 2 - Synchronize Videos
+  stage 3 - Camera Calibration
+  stage 4 - 2d Tracking and 3d Calibration
+  stage 5 - Create output files (using Blender)
+  stage 6 - Create output animation (Matplotlib)
+  ```  
+- `useMediaPipe`
+  - [Type] - BOOL
+  - [Default] - False, 
+  - Whether or not to use the MediaPipe tracking method in `stage=4`
+
+- `runMediaPipe`
+  -	[Type] - BOOL
+  - [Default] - False, 
+  - Whether or not to RUN the MediaPipe tracking method in `stage=4`  (will use previously processed data. This can save a lot of time when re-processing long videos) 
+    
+- `useOpenPose`
+  -	[Type] - BOOL
+  - [Default] - False, 
+  - Whether or not to use the OpenPose tracking method in `stage=4`
+
+- `runOpenPose`
+  -	[Type] - BOOL
+  - [Default] - False, 
+  - Whether or not to RUN the OpenPose tracking method in `stage=4`  (will use previously processed data. This can save a lot of time when re-processing long videos) 
+  
+-  `useDeepLabCut`
+    - [Type] - BOOL
+    - [Default] - False, 
+    - Whether or not to use the DeepLabCut model/project specified at `dlcConfigPath`  to track objects in `stage=4`
+
+
+-  `setDataPath`
+	-	[Type] - BOOL
+   - [Default] - False, 
+   - Trigger the GUI that prompts user to specify location of `FreeMoCap_Data`
+
+-  `userDataPath`
+	-	[Type] - BOOL
+     - [Default] - False, 
+     - path to the location of `FreeMoCap_Data`
+
+-  `recordVid`
+	-	[Type] - BOOL
+     - [Default] - False, 
+     - wehether to save the matplotlib animation to an `.mp4` file
+
+-  `showAnimation`
```

Comment:
I think `showAnimation` is just for whether it will show the actual matplotlib animation at the end of the pipeline - it will still save out an animation if it's false